tests/test_forward_mode.py

```diff
@@ -1,10 +1,10 @@
 import numpy as np
 
-from GrADim.forward_mode import ForwardMode
-from GrADim.GrADim import Gradim
+from GrADim.forward_mode import *
```

Comment:
It is usually a good practice to use the class names and not * to avoid having several objects with similar names. Was it necessary to switch here ?

Reply:
Great! I've changed them just now.